### ISSUE TITLE
instance.ListImages method do not returns public images by default

### ISSUE THREAD

**Issue by QuentinBrosse** [scaleway/scaleway-sdk-go#206]
The SDK uses the default organization ID to fill the `organization` query param and set the `public` query param to `true` of the `/instance/v1/zones/fr-par-1/images` endpoint.
```
GET /instance/v1/zones/fr-par-1/images?organization={ORG_ID}&public=true
```

Therefore, the API returns all the public images from the user default organization. The default behavior that we want for this method is to return all the public Scaleway images.

We need to make this call by default:
```
GET /instance/v1/zones/fr-par-1/images
```

**Comment by QuentinBrosse** [comment]
:wave: @tboerger

**Comment by QuentinBrosse** [comment]
Fixed with #208 #209 and #216.

Thanks again for repporting this issue @tboerger ! 🙏
